pkg/scheduler/scheduler.go

```diff
@@ -86,19 +103,58 @@ func (pc *Scheduler) runOnce() {
 
 func (pc *Scheduler) loadSchedulerConf() {
 	var err error
+	pc.once.Do(func() {
```

Comment:
Why `once`, which make it call only once?

Reply:
Yeah, this is the default configuration, we need load it only once, and if user provides a config file, then load the provided one. If the provoded one is invalid, fallback to the default one and log warning.

Reply:
Ok, that makes sense to me :)